batch/2.0/_index.md

```diff
@@ -1,23 +1,25 @@
 ---
-title: "Jakarta Batch 2.0 (under development)"
-date: 2020-01-15
+title: "Jakarta Batch 2.0 (milestone)"
+date: 2020-06-09
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Batch specifies a Java API plus an XML-based job specification language (JSL), which lets you compose batch jobs in XML from reusable Java application artifacts and conveniently parameterize different executions of a single job.
 
 * [Jakarta Batch 2.0 Release Record]()
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Batch 2.0 Specification Document]() (PDF)
-* [Jakarta Batch 2.0 Specification Document]() (HTML)
+* [Jakarta Batch 2.0 Specification Document](./batch-spec-2.0-draft.pdf) (PDF)
+* [Jakarta Batch 2.0 Specification Document](./batch-spec-2.0-draft.html) (HTML)
 * [Jakarta Batch 2.0 Javadoc](./apidocs)
-* [Jakarta Batch 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Batch 2.0 TCK]()([sig](),[sha](),[pub]()):
+
+https://download.eclipse.org/jakartabatch/tck/eftl/jakarta.batch.official.tck-2.0.0-M1.zip
```

Comment:
Are you trying to associate this url with the TCK line?  You could just put the url within the parens ().  This extra line as currently coded throws off the "Maven coordinates" bullet formatting.

Not sure if you knew of this feature or not, but there is an automatic deploy process that allows you to view the rendered pages.  This is posted to the PR:  https://deploy-preview-182--jakartaee-specifications.netlify.app/